Lib/test/test_asyncio/test_pep492.py

```diff
@@ -125,6 +126,7 @@ def foo(): yield
         self.assertFalse(asyncio.iscoroutine(foo()))
 
 
+    @ignore_warnings(category=DeprecationWarning)
```

Comment:
For this test instead of ignoring warning, you should test that it is actually raised.

Reply:
apologize, I will be more mindful in the future

Reply:
It was necessary, I delete this, but the test failed
https://github.com/python/cpython/actions/runs/10335223756/job/28609524103#step:15:1

Reply:
I have fixed it myself, btw you should not mark comments as resolved if they aren't.